p2panda-rs/src/schema/schema_fields.rs

```diff
@@ -0,0 +1,142 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+use std::collections::btree_map::Iter;
+use std::collections::BTreeMap;
+
+use super::error::SchemaFieldError;
+use super::validate::validate_field_name;
+use super::FieldType;
```

Comment:
Let's not use super when it's not somewhat required because of visibility or tests

Reply:
👍